### PR TITLE
Add Fingerroot activation config

### PR DESCRIPTION
**Pull request by marcos-iov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,12 +21,9 @@ npm i
...
 ```
 
-## Run Sample Code
+## Run sample code
 
-Once you do this you can use it as the sample shows in [here](samples/sample.js).
-```
-node sample/sample.js
-```
+There is a sample file included [here](samples/sample.ts).
```
Comment (kelvinator07) [inline README.md:26]: Command To Run The Sample File?
Maybe The Command To Run The Tests Can Also Be Helpful

Comment (marcos-iov) [reply README.md:26]: I prefer not to include any reference to any external library here. I'll let the users decide how to run the sample

Comment (nathanieliov) [reply README.md:26]: So what about changing the header?

`## Sample code`

**Review by nathanieliov — APPROVED** [review]
LGTM.

Just an suggestion.

File: README.md (hunk 1 above)
Comment (marcos-iov) [reply README.md]: Makes sense, thanks

**Pull request merged**